ts/splitter.ts

```diff
@@ -155,9 +155,23 @@ export default class Splitter {
       }
     }
 
+    this.sortDependencies(deps);
+
     return deps;
   }
 
+  private sortDependencies(deps: Map<string, BundleDependencies>) {
+    for (const bundle of deps.values()) {
+      this.sortBundle(bundle);
+    }
+  }
+
+  private sortBundle(bundle: BundleDependencies) {
+    for (const imports of values(bundle)) {
+      imports.sort((a, b) => a.specifier.localeCompare(b.specifier));
```

Comment:
As https://github.com/nodejs/node/issues/4067 shows `localeCompare` has already been available pre `6.x`, so we're fine using that here. 😊